lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -43,52 +41,29 @@ class DetailsRenderer {
   }
 
   /**
-   * @param {DetailsJSON|OpportunityDetails} details
+   * @param {LH.Audit.Details} details
```

Comment:
Add a comment for what this render function does.  i.e. render the top level categories of details.